GSoC-2025/ethology: Low-shot detection for semi-automatic labelling (Rajat Kriplani).md

```diff
@@ -0,0 +1,108 @@
+
+### Motivation: Why this project?
+
+This project uniquely aligns my Psychology background with my technical interests in ML. Addressing the annotation bottleneck in ethology using low-shot detection resonates strongly with my understanding of behavioral research challenges. I am motivated by the technical CV problem itself, the chance to contribute to impactful open-source neuroscience tools, the opportunity to learn Napari plugin development, and the end-to-end nature of building both backend logic and a user-facing interface.
+
+This work directly supports my goal of pursuing interdisciplinary projects where ML aids scientific discovery, particularly in behavioral sciences. Gaining practical experience in low-shot learning, applied CV, and scientific tool-building is key to my intended career path.
+
+### Match: Why you?
+
+My blend of technical skills, relevant academic context, and strong motivation makes me a good fit. Proven Python ability, foundational ML knowledge (TensorFlow, learning PyTorch), and practical open-source experience (`ethology`, Liquid Galaxy, Hacktoberfest) demonstrate my capacity to execute the project's technical aspects. My current PyTorch focus directly targets required skills (`nn.Module`, `torchvision`).
+
+My Psychology studies offer crucial user empathy, enabling me to contribute to an intuitive Napari widget design that serves researchers effectively. This unique perspective, combined with an interest in full-stack development (backend models, frontend UI) and my passion for accessible science, positions me well to deliver a valuable tool for the `ethology` community while advancing my own interdisciplinary skills.
+
+### Availability
+
+I confirm I have no other significant commitments (coursework, jobs, internships, vacations) during the GSoC 2025 coding period (approx. May-August). I am prepared to fully dedicate the required time (approx. 30 hours/week) to this project.
+
+---
+
+## GSoC
+
+### GSoC Experience
+
+My primary expectations from the GSoC program are to gain invaluable real-world software development experience under the guidance of experienced mentors. I look forward to contributing code to a meaningful open-source project that has a direct impact on scientific research. I also expect to significantly improve my technical skills, particularly in PyTorch, computer vision, Napari development, and collaborative software engineering practices. Furthermore, I am excited by the prospect of helping to make advanced technology more accessible to researchers from diverse backgrounds.
+
+### Other Applications
+
+Yes, I am also applying to a project with the Liquid Galaxy organization for GSoC 2025.
```

Comment:
That sounds great! but note that only one project proposal will be accepted, see [here](https://summerofcode.withgoogle.com/rules#:~:text=Each%20GSoC%20Contributor%20may%20submit%20up%20to%20three%20(3)%20Project%20Proposals%3B%20however%2C%20only%20one%20(1)%20Project%20Proposal%20may%20be%20accepted%20per%20GSoC%20Contributor.)